examples/ksonnet-example.jsonnet

```diff
@@ -1,9 +1,7 @@
 ((import 'kube-prometheus/main.libsonnet') + {
-   nodeExporter+: {
-     daemonset+: {
-       metadata+: {
-         namespace: 'my-custom-namespace',
-       },
+   values+:: {
+     nodeExporter+: {
+       namespace: 'my-custom-namespace',
      },
    },
  }).nodeExporter.daemonset
```

Comment:
Can we remove this example and amend the doc about `Prometheus Object Name` to be more generic and include changing name and namespace for objects? The mechanism is the same for all objects not only Prometheus.